File: .tmp_py4013/batch_04_sources/2529_maximum_count_of_positive_integer_and_negative_integer.js
// LeetCode 2529 - Maximum Count of Positive Integer and Negative Integer
// https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/

/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumCount = function(nums) {
    let pos = 0, neg = 0;
    for (const x of nums) {
        if (x > 0) pos++;
        else if (x < 0) neg++;
    }
    return Math.max(pos, neg);
};

File: .tmp_rs4013/rust_check09/2529_maximum_count_of_positive_integer_and_negative_integer.rs
struct Solution;

// LeetCode 2529 - Maximum Count of Positive Integer and Negative Integer
// https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/

impl Solution {
    pub fn maximum_count(nums: Vec<i32>) -> i32 {
        let mut pos = 0;
        let mut neg = 0;
        for x in nums {
            if x > 0 {
                pos += 1;
            } else if x < 0 {
                neg += 1;
            }
        }
        pos.max(neg)
    }
}

fn main() {}
